include/items.h

```diff
@@ -1,388 +1,389 @@
 #ifndef GUARD_ITEMS_H
 #define GUARD_ITEMS_H
 
-enum {
```

Comment:
It aligns comments to each block of comments but it should align to the whole block of code.